lib/matplotlib/collections.py

```diff
@@ -1262,62 +1262,60 @@ def __init__(self,
+            The orientation of the **collection** (the event bars are along
+            the orthogonal direction). Defaults to 'horizontal' if not
+            specified or None.
 
-        *lineoffset*
-            a single numerical value, corresponding to the offset of the center
-            of the markers from the origin
+        lineoffset : scalar, optional. Default is 0.
+            The offset of the center of the markers from the origin.
 
-        *linelength*
-            a single numerical value, corresponding to the total height of the
-            marker (i.e. the marker stretches from lineoffset+linelength/2 to
-            lineoffset-linelength/2).  Defaults to 1
+        linelength : scalar, optional. Default is 1.
+            The total height of the marker (i.e. the marker stretches from
+            ``lineoffset + linelength/2`` to ``lineoffset - linelength/2``).
 
-        *linewidth*
-            a single numerical value
+        linewidth : scalar or None, optional. Default is None.
+            If it is None, defaults to its rcParams setting, in sequence form.
 
-        *color*
-            must be a sequence of RGBA tuples (e.g., arbitrary color
-            strings, etc, not allowed).
+        color : color, sequence of colors or None, optional. Default is None.
+            If it is None, defaults to its rcParams setting, in sequence form.
 
-        *linestyle* [ 'solid' | 'dashed' | 'dashdot' | 'dotted' ]
+        linestyle : str or tuple, optional. Default is 'solid'.
+            If it is a string, it has to be among [ 'solid' | 'dashed' |
+            'dashdot' | 'dotted' | '-' | '--' | '-.' | ':' ]. If it is a dash
+            tuple, it has to be of the form::
 
-        *antialiased*
-            1 or 2
+                (offset, onoffseq),
 
-        If *linewidth*, *color*, or *antialiased* is None, they
-        default to their rcParams setting, in sequence form.
+            where *onoffseq* is an even length tuple of on and off ink
+            in points.
 
-        *norm*
-            None (optional for :class:`matplotlib.cm.ScalarMappable`)
-        *cmap*
-            None (optional for :class:`matplotlib.cm.ScalarMappable`)
+        antialiased : [None (default) | 1 | 2], optional.
+            If it is None, defaults to its rcParams setting, in sequence form.
 
-        *pickradius* is the tolerance for mouse clicks picking a line.
-        The default is 5 pt.
+        Additional kwargs of :class:`~matplotlib.collections.LineCollection`
+        can also be used:
 
-        The use of :class:`~matplotlib.cm.ScalarMappable` is optional.
-        If the :class:`~matplotlib.cm.ScalarMappable` array
-        :attr:`~matplotlib.cm.ScalarMappable._A` is not None (i.e., a call to
-        :meth:`~matplotlib.cm.ScalarMappable.set_array` has been made), at
-        draw time a call to scalar mappable will be made to set the colors.
+        %(LineCollection)s
```

Comment:
fc212ee replaces it with the second solution that I mentioned. If it is too verbose, I could change it for the first proposition and squash/rebase the commits.